cmd/cri-containerd/cri_containerd.go

```diff
@@ -48,6 +48,9 @@ func main() {
 	if err != nil {
 		glog.Exitf("Failed to create CRI containerd service %+v: %v", o, err)
 	}
+	if err := service.Start(); err != nil {
```

Comment:
Since travis is overloaded, it takes a long time to test a PR.

I'll split this into another commit.